vllm/config.py

```diff
@@ -876,8 +881,9 @@ def __init__(
         draft_model_config: ModelConfig,
         draft_parallel_config: ParallelConfig,
         num_speculative_tokens: int,
-        ngram_prompt_lookup_max: int,
-        ngram_prompt_lookup_min: int,
+        speculative_disable_queue_size: Optional[int],
```

Comment:
We should validate that this arg is in valid range (I think anything <=1 is invalid?)